libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -293,7 +216,7 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
                 return;
             }
 
-            if (!SlackHelper.VerifySignature(_options.ClientSigningSecret, request, body))
+            if (!SlackHelper.VerifySignature(_slackClient.Options.ClientSigningSecret, request, body))
```

Comment:
Probably VerifySignature method belongs to the wrapper class too. It doesn't call any API method but it's totally related to it.

Reply:
ok